Prevents error:
libpng-config: --libdir option is disabled in Debian/Ubuntu

--- a/setup.py
+++ b/setup.py
@@ -204,7 +204,7 @@ def make_C_extension():
 def find_libpng():
     # Returns (found, include dir, library dir, library name)
     if sys.platform in ("linux", "darwin", "aix"):
-        libpng_config = shutil.which("libpng-config")
+        libpng_config = "@TERMUX_PREFIX@/bin/libpng-config"
         if libpng_config is None:
             warnings.warn("libpng-config not found")
             return False, None, None, None
